<launch>
	<arg name="android_node" value="0"/>

	<node name="twistToMotors" pkg="chefbot_bringup" type="twist_to_motors" output="screen">
    <remap from="cmd_vel" to="cmd_navigation"/>
  </node>

  <node name="base_controller_node" pkg="chefbot_bringup" type="base_controller">
			<param name="publish_tf" value="true" />
			<param name="use_imu" value="false" />
      <param name="publish_rate" value="10.0" />
      <param name="linear_scale_positive" value="0.99" />
      <param name="linear_scale_negative" value="0.99" />
      <param name="angular_scale_positive" value="1.0" />
      <param name="angular_scale_negative" value="1.0" />
      <param name="angular_scale_accel" value="0.0" />
      <param name="alpha" value="0.5" />
    <!--
      if (dth > 0) dth *= angular_scale_positive;
    if (dth < 0) dth *= angular_scale_negative;
    if (dxy_ave > 0) dxy_ave *= linear_scale_positive;
    if (dxy_ave < 0) dxy_ave *= linear_scale_negative;
  -->
  </node>

	<group if="$(eval android_node == 1)">
		<node name="android_imu_node" pkg="chefbot_bringup" type="gyro.py" output="screen">
	    <param name="host" value="192.168.1.100" />
	    <param name="num_callibration_itrs" value="0" />
	    <param name="debug" value="false" />
	  </node>
	</group>

</launch>
